src/content/docs/develop/actions.mdx

```diff
@@ -0,0 +1,147 @@
+        with:
+          workspaces: './src-tauri -> target'
+
+      - name: install frontend dependencies
+        # If you don't have `beforeBuildCommand` configured you may want to build your frontend here too.
+        run: yarn install # change this to npm or pnpm depending on which one you use.
+
+      - uses: tauri-apps/tauri-action@v0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tagName: app-v__VERSION__ # the action automatically replaces \_\_VERSION\_\_ with the app version.
+          releaseName: 'App v__VERSION__'
+          releaseBody: 'See the assets to download this version and install.'
+          releaseDraft: true
+          prerelease: false
+          args: ${{ matrix.settings.args }}
+```
+
+### Arm Runner Compilation
+The process in which you cross compile is quite similar to version 1, however one small change is
+that the base_image uses `Bookworm` instead of `Bullseye`
+
+In order to compile from source you must install nodejs from a snap as the debian version is too old to be used.
+```yaml
+name: Raspberry Pi compile
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pguyot/arm-runner-action@v2.5.2
+        with:
+          base_image: https://dietpi.com/downloads/images/DietPi_RPi-ARMv8-Bookworm.img.xz
+          cpu: cortex-a53
+          bind_mount_repository: true
+          image_additional_mb: 10240
+          optimize_image: false
+          commands: |
+            # Rust complains (rightly) that $HOME doesn't match eid home
+            export HOME=/root
+            # Workaround to CI worker being stuck on Updating crates.io index
+            export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
+            # Install setup prerequisites
+            apt-get update -y --allow-releaseinfo-change
+            apt-get upgrade -y
+            apt-get autoremove -y
+            apt-get install curl
+            curl https://sh.rustup.rs -sSf | sh -s -- -y
+            . "$HOME/.cargo/env"
+            curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash
+            # Install framework specific packages
+            apt-get install -y snap
+            sudo snap install node --channel=20/stable --classic
+            npm install next@latest react@latest react-dom@latest eslint-config-next@latest
+            # Install build tools and tauri-cli requirements
+            apt-get install -y libwebkit2gtk-4.1-dev build-essential wget libssl-dev libgtk-3-dev libayatana-appindicator3-dev librsvg2-dev
+            # Install frontend dependencies
+            npm install
+            # Build the application
+            npm run tauri build
+      - name: Upload deb bundle
+        uses: actions/upload-artifact@v3
+        with:
+          name: Debian Bundle
+          path: ${{ github.workspace }}src-tauri/target/release/bundle/deb/tauri_1.4_arm64.deb
```

Comment:
It should be possible to build the .deb on an x86_64 runner (like we do with the macos builds above), so maybe it's better to showcase the .appimage here instead (or both)

Reply:
Hmm, no special args should be needed. If you're using the latest tauri versions it should just work 🤔 Did you see any errors?

Reply:
Error in appimage.sh
Also this is done using the standard Ubuntu runner, its just using qemu for a arm64 OS. Its pretty much the v1 docs but with some adjustments and I am planning to add comments for adjustments that need to be made for armhf.

Reply:
The Docker method causes issues with dependencies. Perhaps this will be fixed in the future, but for now it needs to be compiled using this method or a similar method.

Reply:
I had issues with the AppImage too. For some strange reason it tries to execute it or something and since it's inside a Docker it can't do that and therefore fails. That's the reason why I only have `.deb` builds for ARM, and no AppImage. I remembered discussing it somewhere before, but I can't find it now